kernel/async_channel/src/lib.rs

```diff
@@ -246,10 +262,22 @@ impl <T: Send> Sender<T> {
 }
 
 /// The receiver side of a channel.
-#[derive(Clone)]
 pub struct Receiver<T: Send> {
     channel: Arc<Channel<T>>,
 }
+
+impl<T: Send> Receiver<T> {
```

Comment:
forgot the `Clone` trait here

```suggestion
impl<T: Send> Clone for Receiver<T> {
```